tests/sharepoint/resources/list_item3.json

```diff
@@ -32,7 +32,7 @@
   "Weight_x0020_after_x0020_Surgery": null,
   "PedigreeName": "Slc6a3-Cre-650102",
   "Breg2Lamb": null,
-  "HeadpostType": "AI Straight Headbar",
+  "HeadpostType": "Mesoscope-style well with NGC-style headframe (0160-200-20/0160-100-10)",
```

Comment:
same question - doesn't type and part number get separated?

Reply:
oh this is the actual data from the SWR rather than the json is creates - is that right?